### ISSUE TITLE
Reference Error in Heroku

### ISSUE THREAD

**Issue by kambleaa007** [HarryH-Tech/computer-researchers#1]
https://github.com/HarryH-Tech/computer-researchers/blob/17f5340eb31a77f0fc6d80b5d880da2d45157209/client/src/context/ResearchContext.js#L78

**Comment by kambleaa007** [comment]
what is the `exports` its undefined

**Comment by HarryH-Tech** [comment]
Yep, exports is undefined. The JS file is generated by the build command, when the compiler converts it from TS to JS it just puts exports.ResearchContext in the file, I don't know which part of the TS code causes this.

Initially this line: 
const ResearchContext = createContext<ResearcherContextType || null>(null); 
was 
export const ResearchContext = createContext<ResearcherContextType || null>(null);

I thought that might be the cause of it but I removed the 'export' and it still results in the same error.

Thanks for commenting 😊
